pythoncalculator/subtract.py

```diff
@@ -0,0 +1,2 @@
+def subtract(x, y):
+    return y - y
```

Comment:
The parameters here created some errors. Do you wanna change to:
```suggestion
    return x - y
```